apps/public/src/app/(map)/locatie/kies/_components/Combobox/Combobox.module.css

```diff
@@ -1,4 +1,6 @@
 .combobox {
+  /* The longest address in Amsterdam is 77 characters  */
+  max-width: calc((var(--ams-text-input-padding-inline) * 2) + 77ch);
```

Comment:
This is a bit strange since this app is voor multiple municipalities.

Reply:
Good point. I've removed it and made a note to discuss this with Linda.